### DISCUSSION TITLE
How can I reverse the order of command history?

### DISCUSSION THREAD

**Discussion by the21st** [PatrickF1/fzf.fish#215] [Q&A] [answered]
I'm used to having the most recent history order at the bottom when using ctrl+r. The default here is at the top. Can I configure the order?

I searched for "order" in the readme and in discussions and found nothing.

**Comment by PatrickF1** [comment, marked as answer]
Pass `--layout=default` using https://github.com/PatrickF1/fzf.fish#pass-fzf-options-for-a-specific-feature or https://github.com/PatrickF1/fzf.fish#always-pass-some-options-to-fzf

**Reply by the21st** [reply]
Thanks! Those readme sections are a bit unclear to me, but I ended up editing the `_fzf_wrapper.fish` file and I deleted the string `--layout=reverse` from the default options and this got me what I wanted.
